Gaijinizer patch/Scripts.txt

```diff
@@ -2829,90 +2829,85 @@ It is already a limit
-
-And more important things ...
-The 'genre' of that device.
-
-Why did you start this device?
-Of course, I think that the underlying motive is dozens of colors
-'Because you were likely to have a favorite state'?
-
- Even the actors are human beings, not gods.
- In order for 'Heiz' to reach the end, you, the intermediator, must first
-We needed the 'ability' to reach the final ending.
-
- It can be enjoyed even if she is committed by a woman demon...but rather, 'M temperament'
- 'Extra love' that can tolerate the appearance of the appearing demon
-
-And above all...'The gut' that can stand up to this difficulty and rise up again and again
-
-If it's just a game, you can weaken the enemy and strengthen the operation character
- But the 'device' is not that good.
-Although correction can be applied to some extent, it is only that.
- In particular, what you changed was 'the destiny of death' which should have been said to be universal
-It is not a play that can be the guy of that day.
-
-In order to identify that “ability”
-A prototype of the intervention device was distributed as a 'trial version'.
-It will be a long time ago, but do you remember the contents?
-That is 'where' and 'what format',
-Did you 'intervene' in that world .........
-Probably because it was still an experimental device
- I wonder how fragmented it was.
-
-... yeah? Is it time?
-Well then, one last thing!
- It's likely to be misunderstood, but with 'a person who has organized this device'
-'The creator of this game' is probably another person.
- The latter only thinks 'I made a niche eroge'.
-
- 'The guy who put in the equipment' is a que do not know who
-It's probably a resident of 'this world' if it's not a mere entertainer.
- The guys at Hell's Gate have only worked with them.
-It changes her world and tries to let you do something.
-
-But if you say that his journey is over safely ...
-This time, that 'preparing'...may be.
-
-
-Now! Anyway, I'm tired!
-You have successfully completed this game!
- Let's make deep reading a pear, and let's be purely pleased there!
-
-...... If I have something to meet in another place
- At that time, let's get along well again!
- It's a promise!
- =================================================="#MTLed
+ 《Body》==========================================
+...I wonder if this'll work.
+Professor Yui said she'd invented this weird device to be able to
+send messages directly to you, the 'Intervener', so I thought I'd give
+it a try. Apparently the messages come to you in the form of some kind of
+text email.
+
+Well first off, I suppose I should congratulate you. Maybe something
+like, 'Congrats on finishing the game!'. That's right, I know what kind
+of device you've been using to carry out your interventions. That's because
+this isn't the first world that's received very similar intervention.
+There was a world that received similar interventions before. And it's
+always someone from your world who's chosen to intervene.
```

Comment:
Some minor flow:
>_**That's because this isn't the first world that this has happened in.
There was another world that received a similar intervention like this before...
And it seems like it's always been someone from your world who's chosen to
intervene.**_